src/weights.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for pallet_crowdloan_rewards
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 3.0.0
+//! DATE: 2021-07-12, STEPS: `[32, ]`, REPEAT: 32, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/moonbeam
+// benchmark
+// --chain
+// dev
+// --execution=wasm
+// --wasm-execution=compiled
+// --pallet
+// pallet_crowdloan_rewards
+// --extrinsic
+// *
+// --steps
+// 32
+// --repeat
+// 32
+// --raw
+// --heap-pages=4096
+// --template=./benchmarking/frame-weight-template.hbs
+// --output
+// /tmp/
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{
+	traits::Get,
+	weights::{constants::RocksDbWeight, Weight},
+};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_crowdloan_rewards.
+pub trait WeightInfo {
+	fn initialize_reward_vec(x: u32, y: u32) -> Weight;
+	fn complete_initialization(x: u32) -> Weight;
+	fn claim(x: u32) -> Weight;
+	fn update_reward_address(x: u32) -> Weight;
+	fn associate_native_identity(x: u32) -> Weight;
+}
+
+/// Weights for pallet_crowdloan_rewards using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	fn initialize_reward_vec(x: u32, _y: u32) -> Weight {
+		(22_062_914_000 as Weight)
```

Comment:
seems really high but maybe that is expected

@notlesh told me that the second time benchmarking is run on the machine produces significantly lower weights consistently

Reply:
yay it's so much lower now!